vendor/engines/secure_rooms/config/routes.rb

```diff
@@ -15,7 +15,13 @@
       patch :"card_number/update", to: "secure_rooms/card_numbers#update"
     end
 
-    resources :occupancies, controller: "secure_rooms/facility_occupancies", only: :index
+    resources :occupancies, controller: "secure_rooms/facility_occupancies", only: :index do
+      collection do
+        post "assign_price_policies_to_problem_orders"
```

Comment:
I'm still interested in DRYing this up, but I won't consider a show stopper for now, especially if there are going to be more routes coming up.